.github/workflows/run-zombienet-tests.yml

```diff
@@ -168,3 +175,13 @@ jobs:
         uses: ./.github/workflow-templates/zombienet-tests
         with:
           test_name: ${{ matrix.test_name }}
+
+      - name: Create issue with gh CLI
+        if: failure()
+        run: |
+          gh issue create \
+            --title "🚨 Critical Test Failure in ${{ github.workflow }}" \
```

Comment:
I am not sure I want to label this as critical. we know how zombienet tests are, they are not as reliable. I am not sure we win much from this basically, considering most of the zombienet tests run alreayd in PRs no?

Reply:
We want to exclude some zombienet tests from PR CI and put it to here, like for example warp sync.